content/en/security/application_security/enabling/compatibility/nodejs.md

```diff
@@ -0,0 +1,48 @@
+---
+title: NodeJS Compatibility Requirements 
```

Comment:
```suggestion
title: Node.js Compatibility Requirements 
```

At the risk of opening a massive can of worms... have we standardized how we render "Node.js" (versus nodejs, NodeJS, etc.?) As far as I can tell, `Node.js` is how it's rendered by the developer (OpenJS Foundation) and the trademark is registered as such — [https://openjsf.org/wp-content/uploads/sites/84/2022/02/OpenJS-Foundation-Trademark-List-2022-02-14-1.pdf](https://openjsf.org/wp-content/uploads/sites/84/2022/02/OpenJS-Foundation-Trademark-List-2022-02-14-1.pdf)